### DISCUSSION TITLE
Reorder Line Items when creating/editing CP Orders

### DISCUSSION THREAD

**Discussion by BrandKits** [craftcms/commerce#2792] [Ideas]
Craft the perfect CP order... forget an item, and it automatically appears at the top when added, with no way to reorder items, jumbling the order, and making little sense to the customer.

We have clients making at least 50% of orders in the CP, and not being able to reorder line items is fast becoming a problem.

Can Order > Line Items have the same controls as Categories/Structures, reflecting the order on the back and front ends?

Thanks

**Comment by lukeholder** [comment]
Closing as duplicate of for #3597, please discuss over there, thanks!
